go-controller/pkg/libovsdbops/router.go

```diff
@@ -997,16 +997,16 @@ func FindNATsWithPredicate(nbClient libovsdbclient.Client, predicate natPredicat
 // GetRouterNATs looks up NATs associated to the provided logical router from
 // the cache
 func GetRouterNATs(nbClient libovsdbclient.Client, router *nbdb.LogicalRouter) ([]*nbdb.NAT, error) {
-	router, err := GetLogicalRouter(nbClient, router)
+	r, err := GetLogicalRouter(nbClient, router)
```

Comment:
`router` has too many characters for you? :)

Reply:
heh its cause the router variable overrides the router param to the function, so if we get an error here and I try to do my new fmt.Errorf where I print the router.name, router  is nil and we get segfault :)

Reply:
ah, sorry I missed that. thanks for catching this, Tim!